community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/command/Command.java

```diff
@@ -150,122 +137,87 @@ public String toString()
             return beforeAndAfterToString( before, after );
         }
 
-        public NodeRecord getBefore()
+        public RECORD getBefore()
         {
             return before;
         }
 
-        public NodeRecord getAfter()
+        public RECORD getAfter()
         {
             return after;
         }
     }
 
-    public static class RelationshipCommand extends Command
+    public static class NodeCommand extends BaseCommand<NodeRecord>
     {
-        private RelationshipRecord record;
-
-        public RelationshipCommand init( RelationshipRecord record )
+        public NodeCommand init( NodeRecord before, NodeRecord after )
```

Comment:
`#init()` methods in all `extends BaseCommand` classes look very similar. can we have put `#init()` in `BaseCommand`?